### ISSUE TITLE
Re#736 - Form Elements, fancy buttons (Pt 3)

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#883]
**Blocked**
There are some comments related to Buttons at backdrop-ops/styleguide#11

**Blocked again**
This issue is also blocked by https://github.com/backdrop/backdrop-issues/issues/882 which will add classes everywhere we need a fancy submit button.

Implement special button styles:
* Primary button styles (blue?)
* Dangerous button styles (red?)
* Other button styles (may appear next to primary or dangerous, and need to still be pretty)

![fancy](https://cloud.githubusercontent.com/assets/397895/7290845/ca467e3e-e93a-11e4-9198-a381ebabb082.png)

**Comment by dariusgarza** [comment]
Latest and greatest button scheme: https://www.dropbox.com/s/27xcd47d6sgd4a9/BackdropCMS-Buttons-01.pdf?dl=0

**Comment by dariusgarza** [comment]
Added a cancel button style if necessary - https://www.dropbox.com/s/27xcd47d6sgd4a9/BackdropCMS-Buttons-01.pdf?dl=0


![backdropcms-buttons-01](https://cloud.githubusercontent.com/assets/7042667/7435342/dc738be2-eff7-11e4-9d0f-4096c18ce0ee.png)

**Comment by quicksketch** [comment]
I think these look great!

So to be clear:

- Blue button: primary form submit button (usually saves data)
- White/Gray button: secondary form submit button (e.g. the old "Preview" on the node form)
- Gray/Green button: generic button (e.g. "Upload" file or "Add another" field)
- White/Red button: primary button on deletion/confirmation forms

**Comment by dariusgarza** [comment]
Yup, that's exactly what I had in mind! :+1:

**Comment by quicksketch** [comment]
These styles have all been implemented as part of https://github.com/backdrop/backdrop-issues/issues/882. Thanks @dariusgarza!

One issue I found is that focused buttons are very difficult to determine. Using the mouse, things are just fine on hover and click, but a focused button (via the keyboard) is very difficult to see. I filed a followup at https://github.com/backdrop/backdrop-issues/issues/909.
